playbooks/roles/edxapp/defaults/main.yml

```
@@ -339,6 +339,9 @@ EDXAPP_FEATURES:
   SHOW_HEADER_LANGUAGE_SELECTOR: "{{ EDXAPP_SHOW_HEADER_LANGUAGE_SELECTOR }}"
   SHOW_FOOTER_LANGUAGE_SELECTOR: "{{ EDXAPP_SHOW_FOOTER_LANGUAGE_SELECTOR }}"
 
+# Disable wiki by default
+EDXAPP_WIKI_ENABLED: false
```

```suggestion
EDXAPP_WIKI_ENABLED: true
```

We need it for Tahoe. Disabling it for specific customers seems like a better way to go.

---

@OmarIthawi Done

---

Thank you!